build.gradle

```diff
@@ -85,91 +86,76 @@ configurations {
 }
 
 dependencies {
+    // legacy dependencies [apple, customjfx, spin]
     compile fileTree(dir: 'lib', includes: ['*.jar'])
 
-    compile 'com.jgoodies:jgoodies-common:1.8.1'
-    compile 'com.jgoodies:jgoodies-forms:1.9.0'
-
-    compile 'org.apache.pdfbox:pdfbox:2.0.9'
-    compile 'org.apache.pdfbox:fontbox:2.0.9'
-    compile 'org.apache.pdfbox:xmpbox:2.0.9'
-
-    // required for reading write-protected PDFs - see https://github.com/JabRef/jabref/pull/942#issuecomment-209252635
```

Comment:
Should I convert the comment to an [ADR](https://adr.github.io/madr/) and link it (in a comment `// see ADR-0004`)? I added that comment, because someone might wonder why bouncycastle is integrated.